### PR TITLE
docs: fix spelling of words

### PR DESCRIPTION
**Pull request by exposir**
Typescript -> TypeScript

### CODE REVIEW

File: cli/dts/typescript.d.ts [hunk 1]
```diff
@@ -1953,7 +1953,7 @@ declare namespace ts {
         readonly parent: JSDoc;
         readonly name: EntityName;
         readonly typeExpression?: JSDocTypeExpression;
-        /** Whether the property name came before the type -- non-standard for JSDoc, but Typescript-like */
+        /** Whether the property name came before the type -- non-standard for JSDoc, but TypeScript-like */
```
Comment (kitsonk) [inline cli/dts/typescript.d.ts:1956]: This is what exists in the upstream TypeScript projects and is copied directly from their distributions.  If you want to fix this, you will need to raise a PR at https://github.com/microsoft/TypeScript.

File: cli/tsc/00_typescript.js [hunk 2]
```diff
@@ -47332,7 +47332,7 @@ var ts;
         }
         function bindSpecialPropertyAssignment(node) {
             var _a;
-            // Class declarations in Typescript do not allow property declarations
+            // Class declarations in TypeScript do not allow property declarations
```
Comment (kitsonk) [inline cli/tsc/00_typescript.js:47335]: This is what exists in the upstream TypeScript projects and is copied directly from their distributions.  If you want to fix this, you will need to raise a PR at https://github.com/microsoft/TypeScript.

File: cli/tsc/00_typescript.js [hunk 3]
```diff
@@ -91955,7 +91955,7 @@ var ts;
                     // Set Accessors can have TypeScript modifiers and type annotations.
                     return visitSetAccessor(node);
                 case 256 /* SyntaxKind.FunctionDeclaration */:
-                    // Typescript function declarations can have modifiers, decorators, and type annotations.
+                    // TypeScript function declarations can have modifiers, decorators, and type annotations.
```
Comment (kitsonk) [inline cli/tsc/00_typescript.js:91958]: This is what exists in the upstream TypeScript projects and is copied directly from their distributions.  If you want to fix this, you will need to raise a PR at https://github.com/microsoft/TypeScript.

**Comment by kitsonk** [comment]
These are all changes to files that are directly from the TypeScript distribution and copied into Deno. A change upstream would automatically get updated when we update TypeScript versions.

File: cli/dts/typescript.d.ts (hunk 1 above)
Comment (exposir) [reply cli/dts/typescript.d.ts:1956]: I understand. Thank you.

**Pull request closed without merging**